CIRG-PROMIS-GLOBAL.json

```diff
@@ -0,0 +1,1481 @@
+            "display": "{score}",
+            "system": "http://unitsofmeasure.org"
+          }
+        }
+      ],
+      "required": false,
+      "linkId": "/71971-6",
+      "text": "PROMIS-10 Global Health, GPH, T score",
+      "item": [
+        {
+          "text": "GPH = Global Physical Health. This score is calculated from question responses via coefficients. Valid range: 16.2 to 67.7.",
+          "type": "display",
+          "linkId": "/71971-6-help",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "text": "Help-Button",
+                "coding": [
+                  {
+                    "code": "help",
+                    "display": "Help-Button",
+                    "system": "http://hl7.org/fhir/questionnaire-item-control"
+                  }
+                ]
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "type": "decimal",
+      "code": [
+        {
+          "code": "71972-4",
+          "display": "PROMIS-10 Global Health, GPH, raw score",
+          "system": "http://loinc.org"
+        }
+      ],
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-unit",
+          "valueCoding": {
+            "code": "{score}",
+            "display": "{score}",
+            "system": "http://unitsofmeasure.org"
+          }
+        }
+      ],
+      "required": false,
+      "linkId": "/71972-4",
+      "text": "PROMIS-10 Global Health, GPH, raw score",
+      "item": [
+        {
+          "text": "GPH = Global Physical Health. This score is obtained from answers to some questions. Valid range: integer 4 to 20.",
+          "type": "display",
+          "linkId": "/71972-4-help",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "text": "Help-Button",
+                "coding": [
+                  {
+                    "code": "help",
+                    "display": "Help-Button",
+                    "system": "http://hl7.org/fhir/questionnaire-item-control"
+                  }
+                ]
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "type": "decimal",
+      "code": [
+        {
+          "code": "85523-9",
+          "display": "PROMIS short form - global - version 1.2 raw score",
+          "system": "http://loinc.org"
+        }
+      ],
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-unit",
+          "valueCoding": {
+            "code": "{score}",
+            "display": "{score}",
+            "system": "http://unitsofmeasure.org"
+          }
+        }
+      ],
+      "required": false,
+      "linkId": "/85523-9",
```

Comment:
@mcjustin I see other scoring items, `PROMIS-10 Global Health, GMH raw score`, `PROMIS-10 Global Health, GPH, T score`, `PROMIS-10 Global Health, GPH, raw score` in addition to this one.  I think we report this one (link Id /95523-9) for PainTracker data.   What are the differences in these scores? I assume they are calculated differently?

Reply:
Yes, I retained the other summary scores that LOINC had, but we're unlikely to use them. You have a typo there, but you're flagging the correct one (85523-9)  :^)